base/docs/basedocs.jl

```diff
@@ -995,6 +996,21 @@ Core.WrappedException
     UndefRefError()
 
 The item or field is not defined for the given object.
+
+# Examples
+```jldoctest
+julia> struct MyType
+           a::Int
+       end
+
+julia> A = MyType(5)
+MyType(5)
+
+julia> getfield(A, b)
+ERROR: UndefVarError: b not defined
+Stacktrace:
+[...]
+```
```

Comment:
This does not examplify a `UndefRefError`.

Reply:
My bad, should have reviewed more carefully.